updateEBSVolumes.sh

```diff
@@ -46,6 +46,15 @@ do
   restoreKeyName=${keyName##*/}
   restoreValue=${nameValue/$backupKeyName/$restoreKeyName}
   aws ec2 create-tags --tags Key=Name,Value=$restoreValue --resources $volumeID --region $region
+
+  #Updaate the volume type
+  pvName="${pv%.*}"
+  claimName=$(kubectl get pv ${pvName} | jq -r '.spec .claimRef .name')
```

Comment:
we need to get the claimName from cluster, as we are removing the it when we run the restoreProvider Script